front-end/src/api/DashboardApi.js

```diff
@@ -1,11 +1,42 @@
 import axios from 'axios';
 
+const url = process.env.NODE_ENV === 'development' ? 'http://localhost:3000' : 'https://githubconnector.herokuapp.com';
+
 export async function pullUserInfo(userID, token) {
-  const url = process.env.NODE_ENV === 'development' ? 'http://localhost:3000' : 'https://githubconnector.herokuapp.com';
+
   return axios.get(`${url}/users/${userID}`,{ headers:{
       'Accept': 'application/json',
       'Content-Type': 'application/json',
       'Authorization': 'Bearer '  + token }
   });
 }
 
+//Attempts to link Github Account
+export async function linkGithubAccount(userID, token, github_username) {
+  const config = {
+    headers:{
+      'Accept': 'application/json',
+      'Content-Type': 'application/json',
+      'Authorization': 'Bearer '  + token }
```

Comment:
Small nitpick but I'd leverage ES6 here and on line 10 to do something like this:

```
'Authorization': `Bearer ${token}`
```